### PR TITLE
MNT: Remove double dependency on dask

### PR DESCRIPTION
**Pull request by hmaarrfk**
- Remove build dep on numpy doc
- Add cytoolz as a dependency for speed
- Update build command

Related to #32

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -37,12 +36,14 @@ requirements:
     - matplotlib >=2.0.0
     - networkx >=1.8
     - pillow >=4.3.0
-    - dask-core >=0.15
```
Comment (jni) [inline recipe/meta.yaml]: Why remove this altogether?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -37,12 +36,14 @@ requirements:
...
     - toolz >=0.7.4
     - pywavelets >=0.4.0
     - dask-core >=0.9.0
     - cloudpickle >=0.2.1
     - imageio >=2.1.0
+    # scikit-image depends on dask-array
+    # which requires numpy, dask-core and toolz (cytoolz for speed)
+    - cytoolz >=0.7.3
```
Comment (jni) [inline recipe/meta.yaml]: I don't know whether it's a good idea include Cytoolz... Doesn't this go against your philosophy of making a minimal package?

Comment (hmaarrfk) [reply recipe/meta.yaml]: I'm mostly for "having choice to create a minimal package if I want to". I don't care for example if `xarray` installs 100 things, but I want to be able to depend on `xarray-core`

1. cytoolz is basically faster toolz.
2. The only dependency ctoolz has is toolz.

conda-forge is about having things work well out of the box. I think cytoolz hits that "just right"

File: recipe/meta.yaml (hunk 1 above)
Comment (hmaarrfk) [reply recipe/meta.yaml]: Oh, in this iteration i decided that 0.9 was the correct version choice ;)

**Comment by hmaarrfk** [comment]
@jni, maybe we should wait for https://github.com/conda-forge/matplotlib-feedstock/pull/178/files to get pulled in so we can depend on `matplotlib-core`?

**Comment by hmaarrfk** [comment]
@jni, doesn't seem so obvious how we would depend on `matplotlib-core >= 2.0.0` without depending on matplotlib `3.0.0`. I suggest we move forward with this initial cleanup while others weigh in on this issue

https://github.com/conda-forge/matplotlib-feedstock/issues/179

**Comment by rth** [comment]
Is anything preventing merging this? It looks good to me FWIW.

**Comment by rth** [comment]
Sure, that not an issue, but I'm hoping one of the other recipes maintainers might be willing to press the green button :) It's a fairly non controversial PR..

**Comment by jni** [comment]
It's not entirely non-controversial to add a new dependency. But whatever, by popular demand, I'm merging. =D

File: recipe/meta.yaml (hunk 2 above)
Comment (jakirkham) [reply recipe/meta.yaml]: This is probably my fault for not reviewing PR ( https://github.com/conda-forge/dask-feedstock/pull/49 ) sooner. I've take a look and it seems basically fine. Maybe that will make things more straightforward here.

**Pull request merged**